week-2/J-array-get-set/exercises2.js

```diff
@@ -7,7 +7,9 @@
 */
 
 var numbers = [2, 2, 3]; // Don't change this array literal declaration
-
+     numbers.push(4);
+     numbers.shift(2);
```

Comment:
This is interesting. How did you learn about shift function? It's good to experiment, but always pay attention to the documentation. shift does not actually take any parameters.

Is there a simpler way to just access and modify the value of an array at a specific index? (in this case, at index 0).